platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/text/LocalGlyphRasterizer.java

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.mapboxsdk.text;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Bitmap;
+import android.graphics.Typeface;
+
+public class LocalGlyphRasterizer {
```

Comment:
Can be package protected I think? Could you please add short JavaDoc description of the class?

Reply:
Added a short JavaDoc description. Made the `drawGlyphMap` static method protected -- is that what you meant?